plugins/google/sql/postgresqlLogDisconnections.js

```diff
@@ -0,0 +1,68 @@
+var async = require('async');
+var helpers = require('../../../helpers/google');
+
+module.exports = {
+    title: 'PostgreSQL Log Disconnections Flag Enabled',
+    category: 'SQL',
+    description: 'Ensures SQL instances for PostgreSQL type have log disconnections flag enabled.',
+    more_info: 'SQL instance for PostgreSQL databases provides log_disconnections flag. It is used to log every attempt to connect to the db server. It is not enabled by default. Enabling it will make sure to log anyone disconnects from the instance.',
```

Comment:
```suggestion
    more_info: 'SQL instance for PostgreSQL databases provides log_disconnections flag. It is used to log every attempt to connect to the DB server. It is not enabled by default. Enabling it will make sure to log anyone who disconnects from the instance.',
```